bin/bundle

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env ruby2.1
```

Comment:
you'll then be at the mercy of whichever ruby the user has installed... do we want that?